### PR TITLE
Bl orient738

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient738.xml [hunk 1]
```diff
@@ -30,20 +32,78 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                         <summary/>
-                        
+                        <msItem xml:id="ms_i1">
+                          <locus from="3r"></locus>
+                          <title type="complete" ref="LIT3171Commen"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <colophon xml:lang="gez" xml:id="coloph1">
+                            <locus target="#140v"></locus>
+                            ተፈጸመ፡ በረድኤተ፡ እግዚአብሔር፡ ወበሥነ፡ ጸጋሁ፡ ወስብሐቲሁ፡ ትርጓሜ፡ አቡነ፡ ዐቢይ፡ በውስተ፡ ቅዱሳን፡
+                            ዮሐንስ፡ አፈ፡ ወርቅ፡ መክእክተ፡ ጳውሎስ፡ ሐዋርያ፡ ኀበ፡ ሰብአ፡ ዕብራውያን፡ ጸሎተ፡ ዝንቱ፡ ልሳነ፡ መዓዛ፡
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient738.xml:48]: ል?

File: LondonBritishLibrary/orient/BLorient738.xml [hunk 2]
```diff
@@ -30,20 +32,78 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                            ጳውሎስ፡ ወረድኤተ፡ ዝንቱ፡ አፈ፡ ወርቅ፡ ዮሐንስ፡ ይዕቀቦ፡ ለኃጥአ፡ ወልዶሙ፡
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient738.xml:49]: እ?

File: LondonBritishLibrary/orient/BLorient738.xml [hunk 3]
```diff
@@ -52,48 +112,54 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">180</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>381</height>
-                                        <width>343</width>
+                                    <measure unit="leaf" type="blank">1</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>15</height>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient738.xml:117]: 150 and 135?

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient738.xml (hunk 1 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient738.xml:48]: sorry, which word do you mean?

File: LondonBritishLibrary/orient/BLorient738.xml (hunk 3 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient738.xml:117]: The measuremnets are in inches

File: LondonBritishLibrary/orient/BLorient738.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply LondonBritishLibrary/orient/BLorient738.xml:48]: መክእክተ፡ (ጳውሎስ፡)

Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient738.xml]: Sure, thank you!

**Review by thea-m — APPROVED** [review]

**Pull request merged**